templates/common/sdk_delete_pre_build_request.go.tpl

```diff
@@ -0,0 +1,3 @@
+    if err = rm.customDeleteModelPackageGroup(ctx, r); err != nil {
```

Comment:
nit: this shouldn't be in common, do you mind renaming this method to requeueUntilCanModify